docker-compose.yml

```diff
@@ -26,12 +28,21 @@ services:
       - SIMCORE_DOCKER_REGISTRY=${SIMCORE_DOCKER_REGISTRY}
       - PUBLIC_NETWORK=${PUBLIC_NETWORK}
       - MONITORED_NETWORK=${MONITORED_NETWORK}
+      - DOCKER_REGISTRY=${SIMCORE_DOCKER_REGISTRY}
```

Comment:
TIP: since you have a single service, I think here it is justified to directly load the .env file instead of rewriting the env vars one by one as `` - VAR = ${VAR}``

See https://docs.docker.com/compose/compose-file/compose-file-v3/#env_file